src/elfpy/wallet.py

```diff
@@ -16,21 +16,23 @@ class Wallet:
 
     # fungible
```

Comment:
Can you add a description of each of these variables in the docstring for this class?